packages/test/test-version-utils/api-report/test-version-utils.api.md

```diff
@@ -81,7 +81,7 @@ export const DataRuntimeApi: {
     dds: {
         SharedCell: typeof cell.SharedCell;
         SharedCounter: typeof counter.SharedCounter;
-        SharedDirectory: typeof map.SharedDirectory;
+        SharedDirectory: ISharedObjectKind<map.ISharedDirectory>;
```

Comment:
The change here seems unnecessary (the typeof statement evaluates to the same thing). I'd rather change the `SharedMap` one back in test-version-utils. I can do that separately though.